docs/semgrep-app/user-management.md

```diff
@@ -1,13 +1,41 @@
 ---
-slug: role-based-access-control
+slug: user-management 
 append_help_link: true
-title: Role-based access control
-description: "This page informs users of role-based access control in Semgrep App and teaches users about roles and role management."
+title: Managing users and roles 
+description: "Learn about roles, user management, and how to implement role-based access control in Semgrep App."
 ---
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# Controlling access through roles
+
+# Managing users and roles
+
+Accounts enable you to manage access to Semgrep resources, such as scans and findings, with varying levels of collaboration and visibility.
+
+Semgrep App has two types of accounts:
+
+<dl>
+    <dt>Personal account</dt>
+    <dd>Every person who signs into Semgrep App is first signed in to a <strong>personal account</strong>. In a personal account, your findings, projects, private rules, and scans are visible only to you.</dd>
+    <dt>Organization account</dt>
+    <dd>To collaborate with others, create an <strong>organization account</strong>. An organization account in Semgrep App requires an existing organization from your source code management (SCM) tool. Creating this type of account integrates Semgrep App into your organization. A user can have many organizations, provided that they are a member of that organization in their SCM.</dd>
+</dl>
+
+Users from the same organization are able to sign into Semgrep App with their SCM's credentials and are automatically added to the organization account. Semgrep App is also able to detect the organization's repositories for scanning.
+
+In organization accounts, collaborators can collectively manage Semgrep App. By default, users are able to: 
+
+* Add projects to scan
+* View and triage findings
+* Determine what rules to run and actions
+* Manage tokens and other settings
+* View an organization's private rules
+
+Semgrep App can restrict features based on user roles. See [Controlling access through roles](#controlling-access-through-roles).
+
+By creating an organization account, teams are able to collaborate on rule writing and the management of repositories. They are able to enforce organization-wide standards and secure their repositories at scale.
+
+## Controlling access through roles
 
 ![Screenshot of role-based access control ](../img/rbac-overview.png)<br />
 
```

Comment:
I could not comment on things outside of your change, but in user-management.md, check these sentences below:

Original:
```md
## Setting up RBAC

Semgrep App divides users into two roles:

* `admin`
* `member`

:::info
Users in organizations without RBAC enabled are `admin` by default.
:::
:::info
Community-tier (Free) users are `admin` by default.
:::
```

Suggesting:
```md
## Setting up role-based access control (RBAC)

Semgrep App divides users into two roles:

* `admin`
* `member`


:::info
Users in organizations without RBAC enabled are assigned an `admin` role by default.
:::

:::info
Community-tier (Free) users are assigned an `admin` role by default.
:::
```

Reply:
(Added in latest commit)